src/containers/AssignmentTexterContact/index.jsx

```diff
@@ -470,23 +470,27 @@ export class AssignmentTexterContact extends React.Component {
       }
     });
 
-    this.tagContact(addedTags, removedTags);
+    this.tagContact(addedContactTags, removedContactTags);
 
     if (callback) {
       this.setState(
-        { addedTags, removedTags, isTagEditorOpen: false },
+        {
+          addedTags: addedContactTags,
```

Comment:
Question: Should we rename the state variables too to make this clear across the board?